src/templates/healthcert/memo/vacV1Memo.tsx

```diff
@@ -49,12 +49,16 @@ const MemoResultSection: React.FC<{
   return (
     <>
       <ResultSection>
-        <p>
-          Based on Singapore&apos;s requirement,{" "}
-          {vaccinationMemoInfo.patientName} has been fully vaccinated against
-          COVID-19.
-        </p>
-        {!multiQr && <p>Thank you.</p>}
+        {!multiQr && (
```

Comment:
Confirm this liner will not be shown for multi QR HealthCerts? Thought it should always be shown 🤔

Reply:
yes, it's in ticket.  i'm just leave this of old single qr healthcert but for multi qr it's look like this
<img width="975" alt="Screenshot 2022-03-08 at 10 50 23 AM" src="https://user-images.githubusercontent.com/10098074/157156729-e9401959-11e5-44a0-a7e7-8d8e3fca2c71.png">